vickev/howdypix
# Watcher

## Description

Watch modification of files in the photos directory: when a file is added, it's analyzed; when a file is removed, it's removed.

Remaining work:

- [x] Catch the exceptions and errors.
- [x] Remove the useless `Thumbnail` entity.





## How to test?

- Check out the branch
- Run the project with `yarn dev` => The files should have the thumbnails created and saved in the database
- Copy/Paste a new file in the `apps/server/photos` directory => The thumbnails should be created.
- Remove this file => The tn should be removed.

## Checklist
